mail/rspamd/src/opnsense/mvc/app/controllers/OPNsense/Rspamd/forms/settings.xml

```diff
@@ -58,6 +58,14 @@
                 <allownew>true</allownew>
                 <help>If an attached file has a suffix in this list, the mail will be rejected via a hard reject, which means that the server will be immeadiately informed about the policy violation.</help>
             </field>
+            <field>
+                <id>rspamd.multimap.whitelistsender</id>
+                <label>Sender Whitelist</label>
+                <type>select_multiple</type>
+                <style>tokenize</style>
+                <allownew>true</allownew>
+                <help>All sender domains listed here are getting whitelisted.</help>
```

Comment:
if this bypasses SPF checks too, please document them here as well as a warning.

Reply:
It will not bypass, the checks will be done, but a -50 points will be added at the end.

Reply:
I am just thinking about enforcing the SPF check to be compliant (for example add the -50 only if the SPF validates correctly, otherwise ignore it to prevent abuse as a "good" sender has an SPF entry in his DNS zone)

Reply:
I can recheck the docs